src/System.Memory/src/System/SpanSortHelpers.Keys.cs

```diff
@@ -0,0 +1,234 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+#if !netstandard
+using Internal.Runtime.CompilerServices;
+#endif
+
+using static System.SpanSortHelpersCommon;
+using S = System.SpanSortHelpersKeys;
+using SC = System.SpanSortHelpersKeys_Comparer;
```

Comment:
Can we use a slightly verbose name? `Helpers_Comparer` maybe.

Reply:
removing them completely